src/api/project.ts

```diff
@@ -7,10 +8,30 @@ const baseURLOject = getDomain();
 const isMock = baseURLOject.MOCK;
 const url = isMock ? project_mock : project;
 
-export function getProjectList(params: any) {
-  return get(url, params).then((res) => res);
+export function getProjectList(params: { page?: number; pageSize?: number }) {
+  return get(url, { params: params }).then((res) => res);
 }
 
-export function createProject(params: any) {
+export function createProject(params: Project) {
```

Comment:
Please redefine the interface for CreateProject and UpdatePorject, Their parameters are different from Project

Reply:
done.